### DISCUSSION TITLE
Grid: React component - Example

### DISCUSSION THREAD

**Discussion by biapar** [carbon-design-system/carbon#12364] [Help]
Are there some example  how to use the Grid and Column React components?
I not understand if the Row component has some role. I try to use but don't work.
I use Carbon v11
Thanks.

**Comment by thiswallz** [comment]
hi @biapar  if I understood it correctly you need some sample for react-grid udner carbon v11, I made a live sample for you, you need to just import the correct libs and import the correct files (including grid sass)

I made a simple example here: [https://codesandbox.io/s/sweet-villani-szidl1?file=/src/App.js](https://codesandbox.io/s/sweet-villani-szidl1?file=/src/App.js)

**Reply by biapar** [reply]
Thanks. Do you use the css class and not Grid, Column React component?

**Comment by tay1orjones** [comment]
@biapar There are various grid examples in the storybook https://react.carbondesignsystem.com/?path=/story/elements-grid--default

**Comment by lSilvani** [comment]
@tay1orjones if I understand correctly, the `Row` element is not needed anymore as we can: 

1. use subgrids
2. rely on the implicit grid 

Am I correct?
